src/lib/components/UserChannel.svelte

```diff
@@ -1,16 +1,16 @@
 <script lang="ts">
     import * as Avatar from '$lib/shadcn/components/ui/avatar/index';
     import { Skeleton } from '$lib/shadcn/components/ui/skeleton/index';
-    import type { UserChannelData } from '$lib/types';
+    import type { PosterData } from '$lib/types';
 
-    export let user: UserChannelData | undefined;
+    export let user: PosterData | undefined;
```

Comment:
nit: `user` is a bit of an inappropriate variable name. It has one property that is a user and another that is a channel. Our `user.user` references throughout this component are a bit strange.

I recognize that this name is actually also inappropriate in **main**, and I simply didn't notice.

Reply:
Overlooked this when updating the type.  Will fix.